packages/platform-ios/src/link/removeFromStaticLibraries.ts

```diff
@@ -15,7 +14,11 @@ import removeFromPbxReferenceProxySection from './removeFromPbxReferenceProxySec
  *
  * Similar to `node-xcode` addStaticLibrary
  */
-export default function removeFromStaticLibraries(project, path, opts) {
+export default function removeFromStaticLibraries(
+  project: any,
+  path: string,
+  opts: {[key: string]: any},
```

Comment:
Will you do a follow up PR to fix all instance of `any`? There's an awful lot of them…

Reply:
This is the plan. We have so many instances of `any` because library that we rely on - `https://www.npmjs.com/package/xcode` - has no typings. I want to write them in the future and make this module better. For now, this PR is big enough and didn't want to include the next big thing.